src/libcrun/container.c

```diff
@@ -1968,21 +1968,34 @@ struct wait_for_process_args
 static int
 wait_for_process (struct wait_for_process_args *args, libcrun_error_t *err)
 {
+  cleanup_channel_fd_pair struct channel_fd_pair *from_terminal = NULL;
+  cleanup_channel_fd_pair struct channel_fd_pair *to_terminal = NULL;
+  int ret, container_exit_code = 0, last_process;
+  cleanup_close int terminal_fd_from = -1;
+  cleanup_close int terminal_fd_to = -1;
+  const size_t max_events = 10;
```

Comment:
Would it make sense to make this a definition with an added comment about what it's for?

Reply:
not sure, I can add a comment if you feel we need it, but this is an arbitrary limit defined only to avoid using the same constant multiple times.

Reply:
No worries. This is fine, too. A #define is just a habit.